.github/workflows/main.yml

```diff
@@ -54,9 +56,45 @@ jobs:
         env:
           ENABLE_SEARCH: 'true'
 
+      - name: Initialize Deployment
+        uses: bobheadxi/deployments@v0.6.2
+        id: start-deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          env: ${{ github.ref_name }}
```

Comment:
Shouldn't this be set to `preview` (for PRs) and `production` (for `main` branch)?

Reply:
yes, the workflow just lacked the respective logic yet